weed/command/scaffold.go

```diff
@@ -176,7 +176,11 @@ database = "seaweedfs"
 
 [elastic7]
 enabled = false
-servers = "http://localhost:9200"
+servers = "http://localhost1:9200,http://localhost2:9200,http://localhost3:9200"
```

Comment:
can this be a list? similar to line 152 in redis addresses.

Reply:
updated.